frontend/public/components/masthead-toolbar.jsx

```diff
@@ -247,7 +248,12 @@ class MastheadToolbarContents_ extends React.Component {
 
   _launchActions = () => {
     const { clusterID, consoleLinks, t } = this.props;
-    const launcherItems = this._getAdditionalLinks(consoleLinks?.data, 'ApplicationMenu');
+    const launcherItems = this._getAdditionalLinks(consoleLinks?.data, 'ApplicationMenu').filter(
+      (item) => {
+        // prevent ACM (Advanced Cluster Management) from appearing in App Launcher as it has moved to perspective switcher
+        return item.metadata.name !== ACM_LINK_ID;
+      },
+    );
```

Comment:
`consoleLinks.data` is already filtered in `_getAdditionalLinks`, we could just add another condition to that filter instead of filtering twice:

```tsx
_getAdditionalLinks(links, type) {
    return _.sortBy(
      _.filter(links, (link) => link.spec.location === type && link.metadata.name !== ACM_LINK_ID),
      'spec.text',
    );
  }
```

Reply:
fixed